acceptance/suites/tests/010-puppetserver-cli/subcommand/gem.rb

```diff
@@ -85,3 +81,20 @@
   assert_match(/SHELL PATH:/, stdout, "PATH expected but not present")
   assert_match(/INSTALLATION DIRECTORY:/, stdout, "GEM_HOME not present")
 end
+
+step "Install hiera-eyaml gem."
+on(master, "#{gem_install} hiera-eyaml --no-ri --no-rdoc")
+
+step "Check that ruby subcommand can load hiera/backend/eyaml from hiera-eyaml"
+on(master, "#{cli} ruby -- -rrubygems -rhiera/backend/eyaml -e 'puts %{OK}'") do
```

Comment:
It's not, no, `--` is only required when dealing with the `lein` commands, not `puppetserver`.  I'll remove them.